relay-event-normalization/src/normalize/processor.rs

```diff
@@ -188,6 +197,22 @@ impl<'a> Processor for NormalizeProcessor<'a> {
 
         Ok(())
     }
+
+    fn process_span(
```

Comment:
Does this now also run for error events? I believe the `TransactionProcessor` prevented that.

Reply:
Yes, it does for errors. Errors aren't supposed to have spans so I thought it'd be convenient to skip that check over being a bit more defensive.